src/java_bytecode/java_class_loader.h

```diff
@@ -51,22 +69,22 @@ class java_class_loadert:public messaget
       std::string class_file_name;
     };
 
-    // class name to index map
+    /// Maps class names to a jar file entry
```

Comment:
Try to work backwards: 1) who is the audience reading this comment (that will likely be different whether it's a doxygen comment or just an in-code one) and 2) what information would they want. The value of a `typedef` lies in re-using a type. Do you want to explain what the type is? The value of the data member are the objects stored in it.

All that said, the comment does not actually add much information (I'm aware you just copied it) here. Is it a surprise that a `std::map` "maps" something? No. Actually the code is broken here as `entryt` is a poor type name.